### PR TITLE
feat: support pagination for member list

### PR DESCRIPTION
**Pull request by QiShaoXuan**
image

Fixed: https://github.com/toeverything/AFFiNE/issues/4217 (by the way)

### CODE REVIEW

File: packages/component/src/components/member-components/invite-modal.tsx [hunk 1]
```diff
@@ -15,49 +15,45 @@ export interface InviteModalProps {
...
-
-  return null;
-  // return (
-  //   <Menu
-  //     trigger="click"
-  //     content={
-  //       <>
-  //         {Object.entries(Permission).map(([permission]) => {
-  //           return (
-  //             <MenuItem
```
Comment (Brooooooklyn) [inline packages/component/src/components/member-components/invite-modal.tsx:36]: remove it

File: apps/server/src/modules/workspaces/resolver.ts [hunk 2]
```diff
@@ -212,15 +211,25 @@ export class WorkspaceResolver {
...
+    @Parent() workspace: WorkspaceType,
+    @Args('skip', { type: () => Int, nullable: true }) skip?: number,
+    @Args('take', { type: () => Int, nullable: true }) take?: number
+  ) {
     const data = await this.prisma.userWorkspacePermission.findMany({
       where: {
         workspaceId: workspace.id,
       },
+      skip,
+      take: take || 8,
```
Comment (Brooooooklyn) [inline apps/server/src/modules/workspaces/resolver.ts:224]: why the default value of take is 10 in frontend and 8 in the backend?

File: packages/component/src/components/member-components/invite-modal.tsx [hunk 3]
```diff
@@ -15,49 +15,45 @@ export interface InviteModalProps {
...
+//         style={{
+//           marginRight: -10,
+//           height: '100%',
+//         }}
+//       >
+//         {currentPermission}
+//       </MenuTrigger>
+//     </Menu>
+//   );
+// };
```
Comment (Brooooooklyn) [inline packages/component/src/components/member-components/invite-modal.tsx:57]: remove it

**Review by himself65 — CHANGES_REQUESTED** [review]
It would be better if you added a test case

Like, register 10 users and add them into a workspace, open the workspace list, and the pagination should work

**Comment by QiShaoXuan** [comment]
> It would be better if you added a test case
> 
> Like, register 10 users and add them into a workspace, open the workspace list, and the pagination should work

Added in e2e test

**Review by himself65 — APPROVED** [review]

**Pull request merged**